app/src/main/java/com/mediinbusan/app/feature/map/MapScreen.kt

```
@@ -973,35 +1239,198 @@ private data class CardEntry(
 )
 
-private fun Place.toCardEntry() =
-    CardEntry(id = id, title = name, subtitle = address, languages = emptyList(), imageUrl = imageUrl)
+private fun distanceFrom(origin: MapPoint, latitude: Double?, longitude: Double?): Double? {
+    if (latitude == null || longitude == null) return null
+    return haversineDistanceMeters(origin.latitude, origin.longitude, latitude, longitude)
+}
+
+/** 1km 미만은 10m 단위 m로, 그 이상은 소수 한 자리 km로 — 숫자만 쓰므로 언어별 문구가 필요 없다. */
+private fun Double.toDistanceLabel(): String =
+    if (this < 1_000.0) "${(this / 10).toInt() * 10}m" else String.format("%.1fkm", this / 1_000.0)
```

detekt (1.23.8)

[warning] String.format("%.1fkm", this / 1_000.0) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)